templates/service/kosher.txt

```diff
@@ -0,0 +1,24 @@
+[hr][center]
+[small]With gods help
+[table][row][cell]Kosher Department[cell][solcrest][cell]Sector Rabbinate[/table][/small]
+[large][b]Kosher Certificate[/b][/large]
+[/center]
+
+Type of Kosher: [b][field][/b]
+We hereby certify the following: [b][field][/b]
+Which is owned by: [b][field][/b]
+Located at: [b][field][/b]
```

Comment:
Swap this with the line above imo
The following:
Located at:
Which is owned by:

Reply:
```suggestion
Owned by: [b][field][/b]
```